### PR TITLE
Create on-silence_hogen-bays.md

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/articles/on-silence_hogen-bays.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+source_url: "https://pwj.shin-ibs.edu/pacific-world-third-series-number-10-fall-2008"
+drive_links:
+  - "https://drive.google.com/file/d/1Da40Ot92Uv05LqDWKHieONqhqtkXwjlj/view?usp=sharing"
+course: sati
+tags:
+  - problems
+  - soto
+  - zen
+year: 2008
+journal: Pacific World: "Journal of the Institute of Buddhist Studies"
```
Comment (buddhist-uni) [inline _content/articles/on-silence_hogen-bays.md:15]: just `journal: pw`

File: _content/articles/on-silence_hogen-bays.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+authors:
+  - "Hogen Bays"
+external_url: "https://www.shin-ibs.edu/documents/pwj3-10/02Bays.pdf"
+source_url: "https://pwj.shin-ibs.edu/pacific-world-third-series-number-10-fall-2008"
+drive_links:
+  - "https://drive.google.com/file/d/1Da40Ot92Uv05LqDWKHieONqhqtkXwjlj/view?usp=sharing"
+course: sati
+tags:
+  - problems
+  - soto
```
Comment (buddhist-uni) [inline _content/articles/on-silence_hogen-bays.md:12]: Let's remove `soto` here

File: _content/articles/on-silence_hogen-bays.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: "On Silence"
+authors:
+  - "Hogen Bays"
+external_url: "https://www.shin-ibs.edu/documents/pwj3-10/02Bays.pdf"
+source_url: "https://pwj.shin-ibs.edu/pacific-world-third-series-number-10-fall-2008"
+drive_links:
+  - "https://drive.google.com/file/d/1Da40Ot92Uv05LqDWKHieONqhqtkXwjlj/view?usp=sharing"
+course: sati
+tags:
```
Comment (buddhist-uni) [inline _content/articles/on-silence_hogen-bays.md:10]: Let's maybe add `hearing` and ...`samatha`?

**Comment by JJM317** [comment]
Ok, I removed Soto and added Samatha and hearing. I tagged Soto since
silence is a big part of the Soto (Caodong) school - an important term for
them is mozhao (silent [mo] illumination/awareness [zhao]. But would that
tag be too limiting sinceZen schools are directly discussed? Also, if we
put samatha, shouldn't we put vipassana? In Chan/Zen zazen (zuochan)  is
seen as simultaneous samatha and vipassana. Even if we take it as in
tandem, I think this article, speaking on Great Silence, would include a
vipassana aspect. This leads me to another thought, I'm not sure what
hearing has to do with this article. I know Bays makes a one-off about
silence behind noises, but the article seems to be more about a
metaphorical silence which is awakening. Just my two-cents in case it is
helpful.

Kindly,
~Joe

**Comment by buddhist-uni** [comment]
My thinking was just that tagging both Zen and Soto is a bit redundant. If you think that this isn't really applicable to Rinzai, then we should switch the tag from Zen to Soto. What do you think? Does this jive with non-Soto Zen?

The hearing tag was a bit of a McGuffin... I imagine some Buddhist coming across this in the (hypothetical, future) `hearing` tag and being delighted to get more than they asked for. I like opportunities to point back to the practice... But yes, it isn't "really" about hearing... Am I being too cute again? 😅

**Comment by JJM317** [comment]
You had me at McGuffin - hearing should definitely stay in. And in the same
spirit, I would keep Zen over Soto. Historically, silence has been a Soto
thing (other schools even used silent illumination as a pejorative for the
Soto school), but this is the 21st and we're all cool with each other, so
let's draw everyone in on this article. Plus, I think I saw Bays
started out in Soto and switched to Rinzai. If he still values silence,
then I'm all in. Soteriology over intellectual history any day ;)

Thanks,
~Joe


--------------------

**Pull request merged**